lifecycle-application/pom.xml

```diff
@@ -10,6 +10,9 @@
     <artifactId>lifecycle-application</artifactId>
     <packaging>jar</packaging>
     <name>Quarkus QE TS: Lifecycle Application</name>
+    <properties>
+        <quarkus.platform.version>9999999-SNAPSHOT</quarkus.platform.version>
```

Comment:
Can you add a comment here to state the purpose of setting this value here?

Reply:
Updated